types/sauronjs/util/insert.d.ts

```diff
@@ -0,0 +1 @@
+export function insert(params: HTMLElement): void;
```

Comment:
From [the source](https://unpkg.com/browse/sauronjs@0.1.2/src/util/insert.js), it looks like this is a callable module.
Therefore, it should be as follows:
```ts
declare function insert(params: HTMLElement): void;
export = insert;
```
What you currently have models `module.exports.insert = insert`, but what the source shows is `module.exports = insert`.

Reply:
Done.